packages/maskbook/src/extension/options-page/DashboardRouters/Contacts.tsx

```diff
@@ -35,7 +35,33 @@ const useStyles = makeStyles((theme) =>
                 marginLeft: theme.spacing(-2),
                 marginRight: theme.spacing(-2),
             },
+            display: 'flex',
         },
+        fixed: {
+            height: '100%',
+            width: '100%',
+            ['@media (max-width: 320px)']: {
```

Comment:
hmmm, this section seems weird cc @lelenei can you explain?

Reply:
The height of the mobile phone screen cannot be the maximum, so the height of the mobile phone is set according to the width

Reply:
Height: '100%' should be deleted

Reply:
On mobile (in fact, when the viewport's width <= 600), the height of the contacts list on the contact page, is computed as 0.

Reply:
I think this page is not adapted for mobile phones, so I just manually calculated the required height.Of course this is incomplete.